django/urls/conf.py

```diff
@@ -67,9 +69,25 @@ def _path(route, view, kwargs=None, name=None, Pattern=None):
             namespace=namespace,
         )
     elif callable(view):
+        # Fail early if the user forgot to call .as_view() on a CBV
+        # (issubclass only accepts types, so guard it using isinstance)
+        if isinstance(view, type) and issubclass(view, View):
```

Comment:
I wonder if [inspect.isclass](https://docs.python.org/3/library/inspect.html#inspect.isclass) wouldn't be clearer here? 🤔